### ISSUE TITLE
Remora .bam as a input

### ISSUE THREAD

**Issue by Fatihlrcfs** [wdecoster/methplotlib#43]
Hi @wdecoster ,

This is not for error/issue. I am just wondering whether can I use the remora .bam file (MM/ML tag) as input.  Also, I could not see the bam input option in the help options. thanks .:)

**Comment by wdecoster** [comment]
Yes, I did implement support for bam,cram with MM/ML tags, but I have to admit that is insufficiently tested. Feedback is therefore appreciated. The file format is recognized, so you don't need a specific input option.

**Comment by Fatihlrcfs** [comment]
Hi @wdecoster ,
I have 2 versions of methplotlib in 2 different env. I used `methplotlib -m age21bc15.sorted2_mapped.bam -n age21freqquency -w 15:23000000-25500000 -g gencode.v39.annotation.gtf.gz` command. The version 0.8.0 got the error? below 
Input file age21bc15.sorted2_mapped.bam not recognized!   File "pandas/_libs/parsers.pyx", line 544, in pandas._libs.parsers.TextReader.__cinit__
  File "pandas/_libs/parsers.pyx", line 633, in pandas._libs.parsers.TextReader._get_header
  File "pandas/_libs/parsers.pyx", li
![age21frequency](https://user-images.githubusercontent.com/75265865/199727739-eea34b1c-9c0d-4da8-8905-9e24a62ed901.JPG)
![age21frequency](https://user-images.githubusercontent.com/75265865/199727813-0bf9d0ce-9f17-4177-a637-e8636b2b8988.JPG)
![age21frequency](https://user-images.githubusercontent.com/75265865/199727861-2e16adc1-7bf3-4704-bf38-fc892853eb40.JPG)
ne 847, in pandas._libs.parsers.TextReader._tokenize_rows
  File "pandas/_libs/parsers.pyx", line 1952, in pandas._libs.parsers.raise_parser_error
UnicodeDecodeError: 'utf-8' codec can't decode byte 0x8b in position 1: invalid start byte
Then ı tried the same command in version 0.20.1 and got some graph below 
Found 0 transcript(s) in the region.
Found 0 transcript(s) in the region.
Found 0 transcript(s) in the region.

**Comment by wdecoster** [comment]
Well it probably wasn't supported yet in the older version.

**Comment by Ge0rges** [comment]
@wdecoster I'm wondering if this has been addressed further as I'm also interested in using what is now the official base caller's output (dorado) with your tool.

**Comment by wdecoster** [comment]
Thanks for reminding me. I will follow up on this, probably next week.
